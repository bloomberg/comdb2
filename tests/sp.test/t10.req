set transaction blocksql
create lua consumer cons_with_sequence_epoch with sequence on (table foraudit for insert and update and delete)
insert into foraudit values(1)
select * from foraudit
exec procedure cons_with_sequence_epoch()
insert into foraudit values(1)
select * from foraudit
exec procedure cons_with_sequence_epoch()
