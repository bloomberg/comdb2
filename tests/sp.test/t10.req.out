(rows inserted=1)
(i=1)
(i=1)
(jsonEmitObj='{"type":"add","sequence":1,"id":"642","epoch":"5678","name":"foraudit","new":"{\"i\":1}"}')
[exec procedure cons_with_sequence_epoch()] failed with rc -3 [string "local function define_emit_columns()..."]:16: attempt to index local 'change' (a nil value)
(rows inserted=1)
(i=1)
(i=1)
(i=1)
(jsonEmitObj='{"type":"add","sequence":2,"id":"642","epoch":"5678","name":"foraudit","new":"{\"i\":1}"}')
[exec procedure cons_with_sequence_epoch()] failed with rc -3 [string "local function define_emit_columns()..."]:16: attempt to index local 'change' (a nil value)
